### PR TITLE
megahit: init at 1.1.4

### PR DESCRIPTION
**Pull request by luispedro**
This is widely used in bioinformatics for assembling genomes &
metagenomes

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)

- Built on platform(s)
   - [ ] NixOS
   - [ ] macOS
   - [x] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [x] Tested execution of all binary files (usually in `./result/bin/`)
Tested main programme, the rest are used internally

- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

File: pkgs/applications/science/biology/megahit/default.nix [hunk 1]
```diff
@@ -0,0 +1,29 @@
+{ stdenv, fetchFromGitHub, zlib }:
+
+stdenv.mkDerivation rec {
+  name    = "megahit-${version}";
+  version = "1.1.3";
```
Comment (markuskowa) [inline pkgs/applications/science/biology/megahit/default.nix]: There is already version 1.1.4 out.

**Comment by markuskowa** [comment]
@GrahamcOfBorg build megahit

File: pkgs/applications/science/biology/megahit/default.nix [hunk 2]
```diff
@@ -0,0 +1,29 @@
+{ stdenv, fetchFromGitHub, zlib }:
+
+stdenv.mkDerivation rec {
+  name    = "megahit-${version}";
```
Comment (markuskowa) [inline pkgs/applications/science/biology/megahit/default.nix]:
```suggestion
  pname    = "megahit";
```

**Comment by luispedro** [comment]
Thanks for the feedback. Corrected & force-pushed the update.

**Pull request merged**